changes/ticket29537

```diff
@@ -0,0 +1,3 @@
+  o Testing:
+    - Check that all valid values of `int` and `unsigned int` can be
+      represented by `void *`. Resolves issue 29537.
```

Comment:
I think this is now "a representative subset of values of int and unsigned int", right?

Reply:
Fixed in 9529d99982eb64998e12db7704fa977bf02f541c.